README.md

```diff
@@ -15,6 +15,8 @@ A React Native app for streaming WMBR 88.1 FM and browsing show archives, song h
 ### Prerequisites
 - Complete the [React Native environment setup](https://reactnative.dev/docs/set-up-your-environment)
 - Install dependencies: `npm install`
+- Install iOS pods: `cd ios && bundle exec pod install`
```

Comment:
whoops, did a clean just now - if you don't have pod installed globally it will fail, but there's a root-level bundle as well:

```suggestion
- Install iOS pods: `bundle install && cd ios && bundle exec pod install`
```

which will also make sure it's the same version/etc